components/closebutton/package.json

```diff
@@ -19,7 +19,7 @@
     "@spectrum-css/vars": "^8.0.0"
   },
   "devDependencies": {
-    "@spectrum-css/component-builder": "^3.2.0",
+    "@spectrum-css/component-builder-simple": "^1.0.0-beta",
     "@spectrum-css/icon": "^3.0.23",
```

Comment:
@yosevu could you also add `"@spectrum-css/tokens": "^1.0.3"` as a `devDependency` and a `peerDependency`?